Python/PDF_Reader/requirements.txt

```diff
@@ -0,0 +1,69 @@
+appdirs==1.4.4

```

Comment:
Use this: https://pypi.org/project/pipdeptree/

It helps to identify what is the dependency tree in your script because I don't think that we need these many packages